pkg/kubelet/cm/dra/manager.go

```diff
@@ -222,14 +224,13 @@ func (m *ManagerImpl) PrepareResources(pod *v1.Pod) error {
 			resourceClaims[claimInfo.ClaimUID] = resourceClaim
 
 			// Loop through all plugins and prepare for calling NodePrepareResources.
-			for _, resourceHandle := range claimInfo.ResourceHandles {
-				claim := &drapb.Claim{
-					Namespace: claimInfo.Namespace,
-					Uid:       string(claimInfo.ClaimUID),
-					Name:      claimInfo.ClaimName,
-				}
-				pluginName := resourceHandle.DriverName
-				batches[pluginName] = append(batches[pluginName], claim)
+			claim := &drapb.Claim{
+				Namespace: claimInfo.Namespace,
+				Uid:       string(claimInfo.ClaimUID),
+				Name:      claimInfo.ClaimName,
+			}
+			for driverName := range claimInfo.Drivers {
+				batches[driverName] = append(batches[driverName], claim)
 			}
```

Comment:
Since we aren't using thw word "plugin" anymore, maybe change the comment to "Loop through all drivers and prepare for calling NodePrepareResources."? Alternatively, reame the local variable to `pluginName` instead of `driverName`...

Reply:
I renamed a few more occurrences of "plugin" with "driver".